cns/restserver/internalapi.go

```diff
@@ -157,13 +157,14 @@ func (service *HTTPRestService) SyncHostNCVersion(ctx context.Context, channelMo
 	if err != nil {
 		logger.Errorf("sync host error %v", err)
 	}
```

Comment:
We dont know here if there is some issue in SyncHostNC API or NMA not able to program a given NC. We need to differentiate it, may be add a error reason.